sig-scalability/configs-and-limits/thresholds.md

```diff
@@ -2,100 +2,72 @@
+| #Services              | 5000                      | 10000                    |
+| #All service endpoints | TBD                       | TBD                      |
+| #Endpoints per service | 250                       | n/a                      |
+| #Secrets               | TBD                       | TBD                      |
+| #ConfigMaps            | TBD                       | TBD                      |
+| #Deployments           | 2000                      | TBD                      |
+| #DaemonSets            | TBD                       | TBD                      |
+| #Jobs                  | TBD                       | TBD                      |
+| #StatefulSets          | TBD                       | TBD                      |
 
+There are also thresholds that depend on environment/cloud provider. The **NOT
+exhaustive** list includes:
 
-The rationale for some of those numbers:
-1. Total number of objects <br/>
-There is a limitation on the total number of objects on the system, as this
-affects among others etcd and its resource consumption.
-1. Number of nodes <br/>
-We believe that having clusters with more than 5000 nodes is not the best
-option and users should consider splitting into multiple clusters. However,
-we may consider bumping the long term goal at some time in the future.
-1. Number of services and endpoints <br/>
-Each service port and each service backend has a corresponding entry in
-iptables. Number of backends of a given service impact the size of the
-`Endpoints` objects, which impacts size of data that is being sent all over
-the system.
-1. Number of objects of a given type per namespace <br/>
-This holds for different objects (pods, secrets, deployments, ...). There are
-a number of control loops in the system that need to iterate over all objects
-in a given namespace as a reaction to some changes in state. Having large
-number of objects of a given type in a single namespace can make those loops
-expensive and slow down processing given state changes.
+| Quantity                         | Threshold scope=namespace | Threshold: scope=cluster |
+|----------------------------------|---------------------------|--------------------------|
+| #Ingresses                       | TBD                       | TBD                      |
+| #PersistentVolumes               | n/a                       | TBD                      |
+| #PersistentVolumeClaims          | TBD                       | TBD                      |
+| #PersistentVolumeClaims per node | TBD                       | TBD                      |
 
----
-<sup>1</sup> The limit for number of pods on a given node is in fact minimum from the “pod per node” and “pods per core times number of cores of a node”.
 
-[Service Level Indicators]: https://en.wikipedia.org/wiki/Service_level_indicator
-[Service Level Objectives]: https://en.wikipedia.org/wiki/Service_level_objective
+[Kubecon talk]: https://www.youtube.com/watch?v=t_Ww6ELKl4Q
```

Comment:
Maybe also add link to the slides - https://docs.google.com/presentation/d/1aWjxpY4YJ4KJQUTqaVHdR4sbhwqDiW30EF4_hGCc-gI/edit?usp=sharing ?

Reply:
Added.